docs/architecture/adr-047-extend-upgrade-plan.md

```diff
@@ -242,9 +200,8 @@ In order to utilize the `UpgradeInstructions` as part of a software upgrade, bot
 
 ## References
 
-* [Current upgrade.proto](https://github.com/cosmos/cosmos-sdk/blob/v0.44.5/proto/cosmos/upgrade/v1beta1/upgrade.proto)
-* [Upgrade Module README](https://github.com/cosmos/cosmos-sdk/blob/v0.44.5/x/upgrade/spec/README.md)
-* [Cosmovisor README](https://github.com/cosmos/cosmos-sdk/blob/cosmovisor/v1.0.0/cosmovisor/README.md)
-* [Pre-upgrade README](https://github.com/cosmos/cosmos-sdk/blob/v0.44.5/docs/migrations/pre-upgrade.md)
-* [Draft/POC PR #10032](https://github.com/cosmos/cosmos-sdk/pull/10032)
+* [Current upgrade.proto](https://github.com/cosmos/cosmos-sdk/blob/main/proto/cosmos/upgrade/v1beta1/upgrade.proto)
+* [Upgrade Module README](https://github.com/cosmos/cosmos-sdk/blob/main/x/upgrade/README.md)
+* [Cosmovisor README](https://github.com/cosmos/cosmos-sdk/blob/main/tools/cosmovisor/README.md)
+* [Pre-upgrade README](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/migrations/01-pre-upgrade.md)
```

Comment:
The `Pre-upgrade README` link is incorrect. The old one still works, but I'm not sure where it was moved to now.

Also, these links need to be to specific version tags and not to `/main/` (or any other branch). They're here to provide context which is lost if the content changes later, or worse, the file moves and the link no longer works.